apps/tangent-electron/src/app/views/creation-rules/CreationRuleEditor.svelte

```diff
@@ -88,10 +130,11 @@ function templateDependencies(template) {
 	{/if}
 	<details>
 		<summary>Name Template Tokens</summary>
-		<p>You can use the following tokens to automate some aspects of note naming.</p>
+		<p>You can use the following tokens to automate some aspects of note naming.<br/>
```

Comment:
It would be better to use two `<p>` tags here, but I actually don't think the text prompting the user to click the buttons is necessary. The combination of the button styling and the tooltips should be sufficient.